app/views/bus_stops/by_sequence.haml

```diff
@@ -0,0 +1,13 @@
+.page-title Route #{@route.number} Bus Stops
+
+.container
+  - @collection.each do |direction, bsrs|
+    = direction
+    - bsrs.each do |bus_stop_route|
+      .row
+        .col-sm-1=bus_stop_route.sequence
```

Comment:
Why is it that we want to show the sequence number here?

Reply:
Also, here and throughout, we have usually stuck to the convention of leaving a space after `-` or `=` in HAML.